eventstore/test_random_contacts.py

```diff
@@ -0,0 +1,134 @@
+from django.test import TestCase, override_settings
+from temba_client.v2 import TembaClient
+
+from eventstore import random_contacts
+import responses
+
+
+class HandleRapidProContactsTests(TestCase):
```

Comment:
We can also move this into test_tasks.py, and rename to be `WhateverTaskNameTests`

Reply:
Do you mean i should move the entire test to test_tasks.py?

Reply:
Yes, we usually have all our tasks in `tasks.py` and all our task related tests in `test_tasks.py`.